### ISSUE TITLE
Error while installing ksonnet bundle

### ISSUE THREAD

**Issue by khaines** [grafana/loki#122]
While following the instructions on https://github.com/grafana/loki/tree/master/ksonnet, I got to the step of installing loki to my cluster. However running the following command in my app directory resulted in an error:

```
jb install github.com/grafana/loki/ksonnet/loki
```
The console output with error:
```
+ jb install github.com/grafana/loki/ksonnet/loki
Cloning into 'vendor/.tmp/jsonnetpkg-loki-master672665173'...
remote: Enumerating objects: 13, done.
remote: Counting objects: 100% (13/13), done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 7301 (delta 4), reused 5 (delta 1), pack-reused 7288
Receiving objects: 100% (7301/7301), 9.34 MiB | 3.45 MiB/s, done.
Resolving deltas: 100% (2771/2771), done.
Already on 'master'
Your branch is up to date with 'origin/master'.
Cloning into 'vendor/.tmp/jsonnetpkg-ksonnet-util-master101695920'...
remote: Enumerating objects: 4872, done.
remote: Total 4872 (delta 0), reused 0 (delta 0), pack-reused 4872
Receiving objects: 100% (4872/4872), 13.41 MiB | 3.41 MiB/s, done.
Resolving deltas: 100% (1429/1429), done.
Already on 'master'
Your branch is up to date with 'origin/master'.
Cloning into 'vendor/.tmp/jsonnetpkg-consul-consul703605583'...
remote: Enumerating objects: 1, done.
remote: Counting objects: 100% (1/1), done.
remote: Total 4875 (delta 0), reused 0 (delta 0), pack-reused 4874
Receiving objects: 100% (4875/4875), 13.41 MiB | 3.61 MiB/s, done.
Resolving deltas: 100% (1432/1432), done.
error: pathspec 'consul' did not match any file(s) known to git
jb: error: failed to install: failed to install package: exit status 1
```

I don't think I missed anything, but please let me know if I can provide any more details.

**Comment by metalmatze** [comment]
I've found the same.
https://github.com/metalmatze/loki-kube-prometheus#current-problems-with-generating

Is this now fixed with #125?

**Comment by tomwilkie** [comment]
Just merged another fix, should work now.  Let me know.
